lib/live_toast/live_component.ex

```diff
@@ -29,6 +29,12 @@ defmodule LiveToast.LiveComponent do
     # If there was, we need to look for a corresponding flash message (with the same kind and message) and remove it.
     sync_toasts = Map.get(assigns, :toasts_sync, [])
 
+    clear_after = Map.get(assigns, :clear_after)
+
+    unless is_nil(clear_after) do
```

Comment:
I wrote `if not` and `Styler` made it `unless`. In newer version unless is deprecated: [[OTP Update and Deprecating Unless](https://elixirforum.com/t/podcast-thinking-elixir-222-otp-update-and-deprecating-unless/66435)](https://elixirforum.com/t/podcast-thinking-elixir-222-otp-update-and-deprecating-unless/66435)